chargeio/services/ServiceBase.cs

```diff
@@ -0,0 +1,14 @@
+﻿using ChargeIO.Infrastructure;
+
+namespace ChargeIo.services
+{
+    public class ServiceBase
+    {
+        protected string SecretKey { get; }
+
+        protected ServiceBase(string secretKey = null)
+        {
+            SecretKey = secretKey ?? Configuration.SecretKey;
```

Comment:
Either this new base service, or Requestor, should be doing this defaulting, but not both.